kubecost/templates/local-store/cluster-storage-pvc.yaml

```
@@ -17,9 +17,13 @@ metadata:
 spec:
   accessModes:
     - ReadWriteOnce
+  {{- if or .Values.localStore.persistentVolume.storageClass .Values.global.defaultStorageClass}}
   {{- if .Values.localStore.persistentVolume.storageClass }}
   storageClassName: {{ .Values.localStore.persistentVolume.storageClass }}
-  {{ end }}
+  {{- else }}
+  storageClassName: {{ .Values.global.defaultStorageClass }}
+  {{- end }}
+  {{- end }}
```

Minor comment. I'm thinking the following would be cleaner and simpler?

```suggestion
  {{- if .Values.localStore.persistentVolume.storageClass }}
  storageClassName: {{ .Values.localStore.persistentVolume.storageClass }}
  {{- else if .Values.global.defaultStorageClass }}
  storageClassName: {{ .Values.global.defaultStorageClass }}
  {{- end }}
```

---

Same idea applies to the `aggregator-statefulset.yaml` template